CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -118,25 +116,66 @@ class SchedulingPlacesViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .backgroundLightGreen
-        navigationController?.navigationBar.isHidden = true // TODO Remove this
 
+        setupViews()
+        setupForStatus()
+        setupConstraints()
+        updateNext()
+    }
+
+    // MARK: - Setup Functions
+    private func setupForStatus() {
+        if schedulingStatus == .choosing {
+            campusLocations = campusLocations.filter { matchLocations.contains($0) }
+            ctownLocations = ctownLocations.filter { matchLocations.contains($0) }
+        }
```

Comment:
I put it before the `locationSections`, because it had to be initialized before the switch in order for `preselectSavedLocations` to not have an index OOB when it tries to select from it. It looks like it should be in the switch though, so I'll move that somewhere where it looks less tricky